API Blueprint Specification.md

```diff
@@ -380,7 +381,7 @@ Furthermore, in API Blueprint context, a payload include its description, descri
 
 A payload **may** have its media type associated. A payload's media type represents the metadata received or sent in the form of a HTTP `Content-Type` header. When specified a payload **should** include nested [Body section](#def-body-section).
 
-This section **should** include at least one of the following nested sections:
+This section **should** include at least one of the following nested sections unless specified otherwise by the sections inheriting this section:
```

Comment:
I think there should be a comma before unless:

> This section **should** include at least one of the following nested sections, unless specified otherwise by the sections inheriting this section: